source/help/settings/channel-settings.rst

```diff
@@ -0,0 +1,59 @@
+Channel Settings
+================
+
+--------------
+
+Notification preferences, channel header, channel purpose, and channel
+name are customizable for each channel. To access these settings, click
+the channel name at the top of the page to open the channel drop-down
```

Comment:
I think I agree with taking out drop-down when we refer to the menus. I think of these as drop-downs, not so much the channel menu or the main menu.
![image](https://cloud.githubusercontent.com/assets/14333569/21336198/f6e0fc5c-c620-11e6-943d-84adacd43b13.png)